platform/metrics/platform-metrics-webconsole-plugin/pom.xml

```diff
@@ -1,141 +0,0 @@
-<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Do metrics work on the admin console without this plugin? Was it specific to the Felix webconsole?